sqlglot/expressions.py

```diff
@@ -2108,6 +2116,11 @@ class If(Func):
     arg_types = {"this": True, "true": True, "false": False}
 
 
+class Ifnull(Func):
```

Comment:
maybe rename this to be IfNull

Reply:
Does camel case imply snake case function? So rename and alias to IFNULL?

Reply:
camel case does imply snake case, but you already overrode the _sqlnames so it should just work

Reply:
```python
class Func(Condition):
    """
    The base class for all function expressions.

    Attributes
        is_var_len_args (bool): if set to True the last argument defined in
            arg_types will be treated as a variable length argument and the
            argument's value will be stored as a list.
        _sql_names (list): determines the SQL name (1st item in the list) and
            aliases (subsequent items) for this function expression. These
            values are used to map this node to a name during parsing as well
            as to provide the function's name during SQL string generation. By
            default the SQL name is set to the expression's class name transformed
            to snake case.
    """
```

Reply:
``` python
   @classmethod
    def sql_names(cls):
        if cls is Func:
            raise NotImplementedError(
                "SQL name is only supported by concrete function implementations"
            )
        if not hasattr(cls, "_sql_names"):
            cls._sql_names = [camel_to_snake_case(cls.__name__)]
        return cls._sql_names
```